src/main/java/sc/iview/commands/edit/add/AddSphere.java

```diff
@@ -60,7 +60,7 @@ public class AddSphere implements Command {
     @Parameter
     private float radius = 1.0f;
 
-    @Parameter
+    @Parameter(required = false)
```

Comment:
So... it's OK for this parameter not to be `lateinit`? I thought @skalarproduktraum said we need to keep `lateinit` on the color parameters? This is currently inconsistent between `AddSphere` here versus `AddBox` and `AddLine`, which use `lateinit`. Surely if *one* of these works a certain way, all three will work that way, and we should be consistent?